FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -130,6 +149,47 @@ const myAuctionConfig = {
 Note that since the `serverResponse` field in the config is a promise it is
 possible to run the on-device auction in parallel with the B&A auction.
 
+## Request Size and Configuration
+
+### Request Size Controls
+The `requestSize` field provided to `navigator.getInterestGroupAdAuctionData()`
+can be used to specify the maximum size of the returned request. If the
+`perBuyerConfig` field is not present (or is empty) then the `requestSize` field
+sets the maximum size for the returned encrypted blob. If the blob fits into a
+size bucket smaller than `requestSize` then that size will be used instead.
+
+If the `perBuyerConfig` field is specified and non-empty, the returned encrypted
+blob will be exactly `requestSize` bytes long unless there was an error. If an error
+occured then the returned blob will be 0 size and the function may throw an error.
+
+### Buyer Controls
+If the `perBuyerConfig` field is specified then the blob will only include the
+interest groups for the buyers specifically listed. If `requestSize` is not
+specified then the sum of the `targetSize` of each buyer will be used for the
+`requestSize`. Interest groups for each buyer will be added to the request -- in
+decreasing order by the interest group's `priority` field -- until the next
+interest group does not fit.
+
+Space can be allocated between different buyers in several different modes:
```

Comment:
How does the browser know which mode to use?

Reply:
It's based on which features of the perBuyerConfig are used across all buyers. The details of when each mode is used are specified with the mode description, below. I clarified this sentence a bit though.